packages/shared/src/components/Feed.tsx

```diff
@@ -139,7 +147,9 @@ export default function Feed<T>({
   const insaneMode = !forceCardMode && listMode;
   const numCards = currentSettings.numCards[spaciness ?? 'eco'];
   const isSquadFeed = feedName === OtherFeedPage.Squad;
-  const { shouldUseMobileFeedLayout } = useFeedLayout();
+  const { shouldUseMobileFeedLayout } = useFeedLayout({
+    originFeedName: feedName,
```

Comment:
I decided not to do it like that as it would require a lot of logic in many places.
Now you only see 1, but the actual is 4 types of feeds which require quite a complex thing, and easy to miss one, vs the hook handling the right case always.

Reply:
Cool, we keep it as is then 🙌 
If I can also suggest we call this feedName and rename in the hook as we do for contextFeedName?